src/ORDataObject/ORDataObject.php

```diff
@@ -5,6 +5,8 @@
  *
  */
 
+namespace OpenEMR\ORDataObject;
+
 class ORDataObject
```

Comment:
when "modernizing" a class, also need to set all class variables and class functions as either public/private